app/src/main/java/org/mifos/mobilebanking/presenters/ThirdPartyTransferPresenter.java

```diff
@@ -107,6 +107,10 @@ public void onNext(AccountOptionAndBeneficiary accountOptionAndBeneficiary) {
                                 getAccountOptionsTemplate());
                         getMvpView().showBeneficiaryList(accountOptionAndBeneficiary.
                                 getBeneficiaryList());
+                        if (accountOptionAndBeneficiary.getBeneficiaryList().isEmpty()) {
```

Comment:
This is not good that we are setting the Beneficiary above then we are chekcing the Beneficiary in below.

Please make the change
```
if(!accountOptionAndBeneficiary.getBeneficiaryList().isEmpty()) {
 getMvpView().showBeneficiaryList(accountOptionAndBeneficiary.getBeneficiaryList());
} else {
 getMvpView().showEmtpyBeneficiary(context.getString(R.string.no_beneficiary_found));
}

Please define `showEmtpyBeneficiary` in ThirdPartyTransferView.